rhombus/private/list.rkt

```diff
@@ -202,6 +380,15 @@
     #`((#%index-result #,(car static-infoss))))
   #'list-build-convert #'())
 
+(define-syntax (treelist-build-convert arg-id build-convert-stxs kws data)
+  #`(for/fold ([lst empty-treelist])
```

Comment:
Will it be worth it to use unsafe operations here?

Reply:
It's difficult to be sure, but my best guess is that it won't matter.